api/cluster-resources/initialize_notebook_cluster.sh

```diff
@@ -18,6 +18,9 @@ jupyter kernelspec uninstall -f pyspark3
 # snippet insertion, see README.md for more details.
 jupyter nbextension enable snippets_menu/main
 
+# Section represents the jupyter page to which the extension will be applied to
+jupyter nbextension enable aou-upload-policy-extension/main --section=tree
```

Comment:
I don't think this line is necessary - did you observe otherwise?

Reply:
you mean the section? yes, if i just pass the script it gets loaded only for the notebook page. In order for it to be loaded/enabled on the tree section it needs to mention the section https://nbgrader.readthedocs.io/en/stable/user_guide/installation.html